spec/paths/enterprise@{enterprise_id}@content@.yaml

```diff
@@ -0,0 +1,90 @@
+get:
+  operationId: getAllContent
+  summary: List content in the enterprise
+  description: Returns Content list
+  produces:
+    - application/json
+  parameters:
```

Comment:
We don't have support for filter's at the moment. However we do have search filter that may search in either name or tags or description.
for eg. `<content_api>/content/?search=test` returns all results that has string test in name, tags or description